src/components/ArticleList.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react'
+import axios from "axios"
+import { ErrorComponent } from './ErrorComponent';
+import { FancyBorder } from './FancyBorder';
+
+export function ArtcileList() {
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        let endpoint = 'https://nc-news-7miy.onrender.com/api/articles'
+        axios.get(endpoint)
```

Comment:
It would be better if the interaction with the api could be separated into another file e.g. `api.js` to keep the component neat.